lisa/operating_system.py

```diff
@@ -150,15 +214,65 @@ def type_name(cls) -> str:
     def name_pattern(cls) -> Pattern[str]:
         return re.compile(f"^{cls.type_name()}$")
 
-    def _install_packages(self, packages: Union[List[str]]) -> None:
+    def _install_packages(
+        self, packages: Union[List[str]], signed: bool = True
+    ) -> None:
+        raise NotImplementedError()
+
+    def _package_exists(self, package: str, signed: bool = True) -> bool:
         raise NotImplementedError()
 
     def _initialize_package_installation(self) -> None:
         # sub os can override it, but it's optional
         pass
 
+    def _get_os_version(self) -> OsVersion:
+        if self._os_version:
```

Comment:
This check logic is in `os_version` already, it doesn't need to check it here.

Reply:
Done